src/platforms/python/guides/fastapi/index.mdx

```diff
@@ -86,6 +86,37 @@ The parameter `traces_sample_rate` needs to be set when initializing the Sentry
 
 </Note>
 
+## Data collected
+
+Here an overview of all the data that is collected by the FastAPI integration
+
+|                                                                       |                                **DEFAULT**                                |                                                                      |                                                                                |                                                                                |
```

Comment:
Is it possible to add a heading to each of the 5 columns? It looks strange to just have one. Also, is it possible to create a table that has lines in it? It would make for a much faster/easier read for the users.

<img width="772" alt="Screenshot 2023-01-27 at 9 21 26 AM" src="https://user-images.githubusercontent.com/26800704/215152706-0ba8df3a-0a91-42a1-9e28-f0a0e81d95e5.png">

Reply:
Hey! Thanks for the review.
Unfortunately the tables are very limited. But I have talked to @Jesse-Box (designer) and he suggested to split the table in two, to make the information more accessible. I am trying this right now.